source/installation/file-location.txt

```diff
@@ -0,0 +1,80 @@
+.. _file-location:
+
+==================================
+Relational Migrator File Locations
+==================================
+
+.. default-domain:: mongodb
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 1
+   :class: singlecol
+
+The following page provides the location of files needed when working with 
```

Comment:
(wording tweak)
If you can, it's a good idea to avoid introductory phrases like "This page ..." or "The following information includes ... " The reasoning here is just that these tend to be throw away phrases. You can often tighten up the intro by omitting them. In this case you could say something like:

```suggestion
Review the location of files needed when working with 
```